code/game/machinery/buttons.dm

```diff
@@ -33,7 +33,7 @@
 /obj/machinery/button/attack_hand(mob/living/user)
 	if(..()) return 1
 	if(istype(user, /mob/living/carbon))
-		playsound(src, "button", 60)
```

Comment:
these are intentionally loud so they can be heard from a decent distance away. 
Don't type in specific numbers at all, use the VOLUME_XXX defines found in defines/audio

Buttons should be high since they're a player initiated action which can often do stuff of significant tactical consequence